kuri_camera/scripts/ros_publisher.py

```diff
@@ -3,18 +3,25 @@
 import threading
 import cv2
 import numpy as np
-from sensor_msgs.msg import Image
+from sensor_msgs.msg import Image, CompressedImage
 from cv_bridge import CvBridge, CvBridgeError
 import rospy
 
 rospy.init_node("kuri_camera_ros_publisher")
 
+# Whether to publish the image msg as a CompressedImage or an Image
+useCompression = True
+
 # We'll treat this like reusable barriers
 image_received = threading.Event()
 image_published = threading.Event()
 
 bridge = CvBridge()
-publisher = rospy.Publisher("/upward_looking_camera/image_raw", Image, queue_size=10)
+baseTopic = "/upward_looking_camera/image_raw"
```

Comment:
Make sure you use [PEP8](https://www.python.org/dev/peps/pep-0008/) casing conventions.

Reply:
fixed -- thanks!